### ISSUE TITLE
Flexi: Track remote gets replaced with project remote

### ISSUE THREAD

**Issue by juschu** [git-moss/DrivenByMoss#373]
I mapped 8 knobs to the 8 track remotes. To test if everything is set up correctly, I moved one of the track remotes I already set up in a certain track to the empty slots for track remotes. As soon as I do that, all the 8 knob suddely control the 8 project remotes instead of the track remotes. When I select the track again I was in, it works as expected again. I don't have to unselect the track before.

Controller: Allen & Heath Xone:K2 (Flexi)
OS: Windows 10
Bitwig: 5.0.11
DrivenByMoss: 21.5.0

**Comment by git-moss** [comment]
Can you elaborate a bit more what you mean by "empty slots for track remotes"? Maybe a screenshot helps.

**Comment by juschu** [comment]
With empty slots I mean the WiFi icons you click to map a new track remote. You can also drag&drop existing track remotes to other slots.

**Comment by git-moss** [comment]
Sorry, I still need more info. You have 8 track parameters and then what? And then you move them to a new page? Where? In the Remote Control Editor? I tried that but did not see any issues.

**Comment by juschu** [comment]
1. I have a track where I already mapped some track remotes, not necessarily 8.
2. I configured my Flexi controller so 8 knobs are set to "Track Remotes: Set Parameter n"
3. I select a track and move the knobs. Those Remotes that are already mapped are working fine. The ones where I still have a WiFi icon obviously don't react because there's nothing to control yet.
4. In the device panel, I drag and drop one of the remotes by grabbing the name to move it to another slot on the same page. Could also be one that is already mapped, so they just switch places.

What I expect: The remote changes its position and thus is now controlled by another knob.
What happens: The 8 knobs now control the remotes of another track. When I just reproduced the error again, it wasn't the master channel but I controlled the remotes of another instrument track.
